src/components/molecules/Visualizer/Engine/Cesium/common.ts

```diff
@@ -278,6 +279,12 @@ export const lookAt = (
   };
 };
 
+export const lookAtWithoutAnime = (scene: Scene) => {
```

Comment:
I kinda love this name haha. But I think, though it will be really long, should be `lookAtWithoutAnimation`.
`Anime` for English speakers is just Japanese animation :P